crm_phonecall/README.rst

```diff
@@ -0,0 +1,59 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+==============
+Crm Phonecalls
+==============
+
+* This module allows to manage phone calls in order to analyze them.
+
+Usage
+=====
+
+To use this module, you need to:
+
+* Go to *Sales > Phone Calls > Logged Calls > Create*.
+* If your user has *Show Scheduled Calls Menu* permission, you will see
+   scheduled calls menu too.
+* In any moment you can schedule another call, schedule a meeting or convert
+   call contact to opportunity.
+* Calls can be categorized and you can manage categories in *Sales >
+   Configuration > Leads & Opportunities > Phone Calls > Categories*.
```

Comment:
Why this option is only available with debub mode? You have to Activate the developer mode to see this... Could you change it please? it doesn't have any sense as a user "never" Activate the developer mode.